ui/raidboss/data/03-hw/raid/a4s.js

```diff
@@ -48,15 +50,18 @@
       condition: function(data) {
         return data.CanStun();
       },
-      alertText: function(data, matches) {
-        return {
-          en: 'Stun ' + matches.target,
-          de: 'Unterbreche ' + matches.target,
-          fr: 'Stun ' + matches.target,
-          ja: 'スタン: ' + matches.target,
-          cn: '眩晕' + matches.target,
-          ko: matches.target + '기절 시키기',
-        };
+      alertText: function(data, matches, output) {
+        return output.text({ param: matches.target });
```

Comment:
needless nit: `param` could be a bit more descriptive

Reply:
Thanks! Changed to `${name}`.